otter/convergence/service.py

```diff
@@ -162,7 +162,7 @@ def all_met(server, current_lb_nodes):
                              if node.matches(server)]))
 
 
-def update_cache(group, servers, lb_nodes, now):
+def update_cache(group, servers, lb_nodes, now, include_deleted=True):
```

Comment:
Hm.. I didn't from that angle. That sounds good. Its fine either way :) I'll leave it upto you.